Makefile

```diff
@@ -2,7 +2,10 @@
 setup:
 	python setup.py develop
 
-test:
+clean:
+	find . -name "*.pyc" -exec rm {} +
```

Comment:
Not great for development. It'll remove any untracked files I'm working on and testing with, but haven't committed yet. That's no fun. I can (and probably should) build this out a bit more to remove all build related files (so `__pycache__/`, `build/`, `dist/`,`htmlcov/`, `smother.egg-info/`, `.cache/`). Did I miss any there?